### ISSUE TITLE
Removing cartopy from dependencies

### ISSUE THREAD

**Issue by w-k-jones** [tobac-project/tobac#287]
Cartopy is currently used in the `plotting` module, as well as being a dependency of `iris`. Installing `cartopy` with `pip` is difficult due to its dependency on the `GEOS` library. If we want to add _`tobac`_ to PyPI we should look into removing `cartopy` from the requirements once we have replaced `iris` with `xarray`. Currently, the only use for `cartopy` is `cartopy.crs.PlateCarree` and `cartopy.mpl.gridliner` which would be better as external options

**Comment by freemansw1** [comment]
I agree with this, but we should keep cartopy as a dependency on the notebooks. 

I've tagged this as a v1.6 item for now, but this could be good for a v1.5.x series release.

**Comment by w-k-jones** [comment]
As of the new cartopy v0.22.0 release, cartopy is easier to install using pip. Hopefully this isn't as much of an issue moving forwards
